apps/bfd-model/bfd-model-rif-samples/src/main/resources/rif-static-samples/sample-mct-update-3-beneficiaries.txt

```diff
@@ -1,3 +1,3 @@
-INSERT|-211|14|999|46250|01-JAN-1910|2|1|0|0|0|10|0|0|{0099190316|FLINTSTONE|FRED||3456789|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|01-Feb-2014|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|||||0|0|0|0|0|0|0|0|||||||||||||||||||||||||S4802|S4802|S4802|S4802|0|0|0|0|0|0|0|0|086|086|086|086|||||||||000|000|000|000|||||||||C|C|C|C|C|C|C|C|C|C|C|C|01|01|01|01|01|01|01|00|00|00|00|00|02|02|02|02|02|02|02|00|00|00|00|00
-INSERT|-212|42|999|35487|01-JAN-1910|2|1|0|0|0|10|0|0|B3499290814|RUBBLE|BETTY||3456789|17-Mar-1981|2019|4|5|6|7|8|A|B|C|D|4|01-Feb-2014|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|H0104|H0104|H0104|H0104|0|0|0|0|0|0|0|0|012|012|012|012|||||||||00|00|00|00|||||||||||||0|0|0|0|0|0|0|0|||||||||||||||||||||||||C|C|C|C|C|C|C|C|C|C|C|C|02|02|02|02|02|02|02|00|00|00|00|00|01|01|01|01|01|01|01|00|00|00|00|00
\ No newline at end of file
+INSERT|-211|14|999|46250|01-JAN-1910|2|1|0|0|0|10|0|0|{0099190316|FLINTSTONE|FRED||3456789|17-Mar-1981|2020|4|5|6|7|8|A|B|C|D|4|01-Feb-2014|5|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|4|4|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|AA|5|C|C|C|C|C|C|C|C|C|C|C|C|D|D|D|D|D|D|D|D|D|D|D|D|||||0|0|0|0|0|0|0|0|||||||||||||||||||||||||S4802|S4802|S4802|S4802|0|0|0|0|0|0|0|0|086|086|086|086|||||||||000|000|000|000|||||||||C|C|C|C|C|C|C|C|C|C|C|C|01|01|01|01|01|01|01|00|00|00|00|00|02|02|02|02|02|02|02|00|00|00|00|00
```

Comment:
Should this now be an `UPDATE`?

Reply:
I wasn't sure because this was modifying an existing file. I don't have insight on how the data is stored or how these files are processed.

Reply:
I'll try and get some answers for you.

Reply:
After some git spelunking I finally have an answer for you.

Could you create a new file, `sample-mct-update-4-beneficiaries.txt`, that contains the all the updates present here? Each row should start with `UPDATE`

Reply:
@whytheplatypus is correct: once loaded, these files are static and shouldn't be changed. Instead, changes should be pushed as new files referencing the same IDs.

Reply:
Would the update rows contain all the data here or only the columns that are changing?

Reply:
I believe all of the data